libs/blocks/merch-card/merch-card.js

```diff
@@ -0,0 +1,200 @@
+/* eslint-disable prefer-destructuring */
+import { decorateButtons } from '../../utils/decorate.js';
+import { loadStyle, getConfig, createTag } from '../../utils/utils.js';
+import { addBackgroundImg, addWrapper, addFooter } from '../card/cardUtils.js';
+import { decorateLinkAnalytics } from '../../martech/attributes.js';
+import { replaceKey } from '../../features/placeholders.js';
+
+const { miloLibs, codeRoot } = getConfig();
+const base = miloLibs || codeRoot;
+
+const SEGMENT_BLADE = 'SegmentBlade';
+const SPECIAL_OFFERS = 'SpecialOffers';
+const PLANS_CARD = 'PlansCard';
+
+const getPodType = (styles) => {
+  const cardTypes = {
+    segment: SEGMENT_BLADE,
+    'special-offers': SPECIAL_OFFERS,
+    plans: PLANS_CARD,
+  };
+  const authoredType = styles?.find((style) => style in cardTypes);
+  return cardTypes[authoredType] || SEGMENT_BLADE;
+};
+
+const createDescription = (rows, cardType) => {
+  const descriptions = rows.slice(0, rows.length - 1);
+  const descriptionWrapper = createTag('div', { class: `consonant-${cardType}-description` });
+  descriptions?.forEach((description) => descriptionWrapper.appendChild(description));
+  return descriptionWrapper;
+};
+
+const createTitle = (titles, cardType) => {
+  const titleWrapper = createTag('div', { class: `consonant-${cardType}-title` });
+  titles?.forEach((title) => titleWrapper.appendChild(title));
+  return titleWrapper;
+};
+
+const decorateFooter = (el, altCtaMetaData, styles, cardType) => {
+  const cardFooter = el.querySelector('.consonant-CardFooter');
+  const replacePlaceHolder = async (key, defaultValue) => {
+    const replacedKey = await replaceKey(key, getConfig(), defaultValue);
+    return { replacedKey };
+  };
+  const decorateWithSecureTransactionSign = () => {
+    const secureTransactionWrapper = createTag('div', { class: 'secure-transaction-wrapper' });
+    const label = createTag('span', { class: 'secure-transaction-label' });
+    const secureElement = createTag('span', { class: 'secure-transaction-icon' });
+    secureTransactionWrapper.append(secureElement, label);
+    replacePlaceHolder('secure-transaction').then(({ replacedKey }) => {
+      label.textContent = replacedKey;
+    });
+    return secureTransactionWrapper;
+  };
+
+  const createCheckbox = (checkBoxText) => {
```

Comment:
the "Add a 30-day free trial of Adobe Stock.*" checkbox is not really supported in the current version, IMO we should remove it.

Reply:
This was a request by GWP/Consonant teams.

Reply:
I guess we can leave it. It's an authorable part and can easily be removed or added to any card.

Reply:
if this must work, the stock OSI value must be added to/removed from cta's `t` atttribute.

see: https://git.corp.adobe.com/wcms/tacocat.js/blob/%40dexter/tacocat-v1.16.1/packages/tacocat/src/checkoutLink.js#L52